samples/bluetooth/iso_broadcast/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Bluetooth ISO Broadcaster
+tests:
+  sample.bluetooth.iso_broadcast:
+    harness: bluetooth
+    platform_allow: qemu_cortex_m3 qemu_x86 nrf52_bsim nrf52dk_nrf52832
```

Comment:
How about the nrf53 actually?

Reply:
It can be added when ISO controller support is functional upstream, as there is need for board specific conf required to match corresponding controller used over rpmsg transport.